controllers/randombeers.controllers.js

```diff
@@ -0,0 +1,3 @@
+exports.renderIndex = (req, res) => {
```

Comment:
This is literally a copy of `home.controller.js`, please remember to update your code immediately when you're copying code otherwise you'll miss/forget some things.

You also need to create a `random-beers.hbs` file.